paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -97,8 +99,10 @@ public class PaparazziPlugin : Plugin<Project> {
       val reportOutputDir = project.extensions.getByType(ReportingExtension::class.java).baseDirectory.dir("paparazzi/${variant.name}")
       val snapshotOutputDir = project.layout.projectDirectory.dir("src/test/snapshots")
 
+      val generateResValuesProvider = project.tasks.named("generate${variantSlug}ResValues") as TaskProvider<GenerateResValues>
```

Comment:
Any better API to retrieve the generated res folder?

Reply:
It seems `InternalArtifactType.GENERATED_RES.[getOutputPath|getIntermediateOutputPath]` does not give the right path

Expected: build/generated/res/resValues/debug
Actual: build/[generated|intermediates]/generated_res/debug

Looking into getOutputPath or getIntermediateOutputPath, what it does inside is just to join paths

```
FileUtils.join(
    getOutputDir(buildDirectory.get().asFile),
    variantIdentifier,
    *paths,
    forceFilename.ifEmpty { getFileSystemLocationName() }
)
```

So it is static and different from actual generated res path. And generated res path could be set via `registerGeneratedResFolders`, so we should always read it from `GenerateResValues` task rather than constructing the static path programatically

Reply:
Oh you're right.  I was confusing the two output directory types, good catch!

We still should try and avoid task references where possible, since that peeks into plugin implementation detail, but not a blocker for this.  Moving to AGP 8 apis, will change this logic to automatically be included as part of localResourceDirs.